.buildkite/release-automation/wheels.rayci.yml

```diff
@@ -8,6 +8,8 @@ steps:
     job_env: forge
     commands:
       - export PYTHON_VERSION={{matrix}}
+      - export RAY_VERSION="$RAY_VERSION"
```

Comment:
is RAY_VERSION set part of the build environment? if so we don't need export it again

Reply:
`RAY_VERSION` and `RAY_HASH` should be put in as environment variables when users create new build on Buildkite. This build, for example, is failing when `RAY_VERSION` is not exported in the step commands: fce7a361807580953364e2da964f9498f3123bf9